_events/2021/2021-04-community-call.md

```diff
@@ -0,0 +1,38 @@
+---
+title: April 2021 Community Call
+expires: 2021-04-08
+event_date: "April 8, 2021"
+layout: event
+repeated: false
+---
+
+The next US-RSE community call is on April 8th at 12ET/11CT/10MT/9PT.
+
+The main topic of discussion is
+**"RSEs in National Labs"**
+
+Charles Ferenbaugh from LANL will be a guest organizer and host a call
+on the topic of [RSEs at National Labs](https://github.com/USRSE/monthly-community-calls/issues/7).
+This call will look at the RSE landscape and practices in what are
+formally known as Federally Funded Research and Development Centers ([FFRDCs](https://www.nsf.gov/statistics/ffrdclist/)).
+These facilities cover all manner of software for technical disciplines including
+weather forecasting, COVID research, stockpile stewardship, 
+national security, energy generation and, of course, software for [landing](https://www.youtube.com/watch?v=4czjS9h4Fpg) 
```

Comment:
This is missing an entire kind of RSE-ing that is not tied to a specific domain, maybe add something like: ?
```suggestion
national security, computer science and systems work, energy generation and, of course, software for [landing](https://www.youtube.com/watch?v=4czjS9h4Fpg) 
```

Reply:
@cferenba would this be okay to add?  It's the kind of work that I do (at a national lab) so I was hoping it could be represented to.

Reply:
Looks good to me - 👍 
I wrote that bit, so its my omission, many apologies to everyone working in that area